packages/lit-dev-content/site/tutorials/content/working-with-lists/02.md

```diff
@@ -0,0 +1,93 @@
+The methods used in the previous steps are great when working with a single
+iterable as a source of data but sometimes the situation might call for a more
+imperative approach.
+
+In this example, we have a string array property `members`, a string array
+property `pets`, a boolean state `includePets` controlled by a button, and
+another boolean state `includeSeparator` also controlled by a button. The
+desired result is to render a list item for each member but also include pets
```

Comment:
The readers YEARN for instruction! suggest changing:

> The desired result is to render a list...

to:

> (insert newline) Render a list... so that...

Maybe even make it a bulleted list of tasks & requirements

also because writing the sentence in the infinitive case ("to render...") leads to weird sentence structure